src/app/components/adminrequests/pending-admin-request/admin-request.component.ts

```diff
@@ -45,12 +47,13 @@ export class AdminRequestComponent {
     query: this.searchQuery,
     approval_status: this.adminStatus,
   };
-  private searchSubject = new Subject<string>();
-
+  searchedQueryNotFound: string = "";
+  searchSubject = new Subject<string>();
```

Comment:
Create a separate service for this subject and use functions to access the service rather than accessing it directly inside the component.